index.html

```diff
@@ -43,18 +43,7 @@ <h2>News</h2>
           <span>(<a id="toggle-news" href="#" role="button">hide</a>)</span>
         </header>
         <ul>
-          <li><time datetime="2020-12-03">3rd December:</time> All times across both event timetables are incorrect since the recent game update. Adam is working on piecing together the new schedules. Please be patient, and let us know what event times you've spotted in the wild via the Google Docs form.</li>
-          <li><time datetime="2020-09-28">28th September:</time>Updated <a href="#role1">Role Event #1</a> and <a href="#role16">Role Event #16</a>.</li>
-          <li><time datetime="2020-09-27">27th September:</time> Updated Legendary Animal Protection events in the role event schedule.</li>
-          <li><time datetime="2020-09-02">2nd September:</time> Free-roam event list has been updated.</li>
-          <li><time datetime="2020-12-03">3rd December:</time> All times across both event timetables are incorrect since the recent game update. Adam is working on piecing together the new schedules. Please be patient, and let us know what event times you've spotted in the wild via the Google Docs form.</li>
-          <li><time datetime="2020-09-02">2nd September:</time> Free-roam event list has been updated.</li>
-          <li><time datetime="2020-08-27">27th August:</time> Sounds like the schedule changed again shortly after my last update. We're working on it. Sorry folks!</li>
-          <li><time datetime="2020-08-24">24th August:</time> Updated free-roam event schedule, once again thanks to <a href="https://www.reddit.com/r/RedDeadOnline/comments/i0n492/new_free_roam_events_schedule/g2o3j4f/">u/outerhaven84</a>.</li>
-          <li><time datetime="2020-08-23">23rd August:</time> Updated <a href="#role1">Role Event #1</a> as it appears to be Day of Reckoning.</li>
-          <li><time datetime="2020-08-21">21st August:</time> I've updated the Role Event schedule, thanks to <a href="https://www.reddit.com/r/RedDeadOnline/comments/i0n492/new_free_roam_events_schedule/g2bal5c/">u/outerhaven84</a>.</li>
-          <li><time datetime="2020-08-19">19th August:</time> There's been another update and the timetable is inaccurate again. If you find a complete list posted with the new times, please <a href="https://github.com/richardwestenra/rdr2-free-roam-event-schedule">let</a> <a href="mailto:richard@richardwestenra.com">me</a> <a href="https://twitter.com/richardwestenra">know</a>.</li>
-          <li><time datetime="2020-08-02">2nd August:</time> I've updated the role event timetable to include the new events from the Naturalist update (thanks to <a href="https://www.reddit.com/r/RedDeadOnline/comments/i0n492/new_free_roam_events_schedule/">Trinerity</a>).</li>
+          <li><time datetime="2020-12-06">December 9th, 2020</time>Schedules have been updated, if you spot an error please submit a correction via the Google Docs form.</li>
```

Comment:
There's a 'hide' button to hide the news, but I've been working on a script to only show the most recent news item unless the user wants to see everything. I'd suggest holding off on deleting everything until that's ready, if that's okay. I've copied this news item into 6149a56